examples/bootstrap.php

```diff
@@ -1,3 +1,7 @@
 <?php
 
 require_once dirname(__FILE__).'/../vendor/autoload.php';
+
+//pull in API key config
+$optionsFile = file_get_contents(dirname(__FILE__).'/example-options.json');
```

Comment:
We'll want to update the script in `composer.json` that creates this file as well.